src/Ginger/Resource.elm

```diff
@@ -170,15 +172,20 @@ Every resource has _one_ category, but can be part of a hierarchy of other
 categories. For example `news` is part of `text > article > news`. This function
 will always return the `Category` stored with the `ResourceWith`.
 
```

Comment:
Every resource has a category, and that category can be part of a category tree. For instance, the `news` category belongs to the category tree `text > news`. This function will return only the category stored with the `ResourceWith`, so in this case `news`, but not its parent `text`.